engine/src/main/java/org/terasology/engine/core/bootstrap/EnvironmentSwitchHandler.java

```diff
@@ -53,10 +60,19 @@
     private PrefabFormat registeredPrefabFormat;
     private PrefabDeltaFormat registeredPrefabDeltaFormat;
 
+    private final Collection<CopyStrategyEntry<?>> typesWithCopyConstructors = ImmutableList.of(
+            new CopyStrategyEntry<>(Quaternionf.class, Quaternionf::new),
+            new CopyStrategyEntry<>(Vector2f.class, Vector2f::new),
+            new CopyStrategyEntry<>(Vector2i.class, Vector2i::new),
+            new CopyStrategyEntry<>(Vector3f.class, Vector3f::new),
+            new CopyStrategyEntry<>(Vector3i.class, Vector3i::new),
+            new CopyStrategyEntry<>(Vector4f.class, Vector4f::new),
+            new CopyStrategyEntry<>(Vector4i.class, Vector4i::new)
+    );
```

Comment:
Is this new, or was this moved from somewhere? 

Edit: I see there's a `RegisterCopyStrategy` annotation that was scanned before. Did we deprecate that in favor of explicitly listing them here?

Reply:
There's a PR for that https://github.com/MovingBlocks/TeraNUI/pull/50